cves/CVE-2013-1965.yml

```diff
@@ -45,7 +45,12 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: |
```

Comment:
What is OGNL? A brief explanation may be nice.

Reply:
OGNL is a simplified expression language for Java.  I can add a bit more of a thorough description to the yaml

Reply:
Agreeing with Matt here. A little bit of an explanation on OGNL would be convenient